content/docs/1.3.0/references/networking.md

```diff
@@ -28,8 +28,9 @@ From | Port | Protocol
 To | Port | Protocol
 --- | --- | ---
 `Other Longhorn Manager` | 9500 | TCP
-`Instance Manager` |  8500; 10000-30000 | TCP
-`Backing Image Manager` |  8000; 8001 | TCP
+`Instance Manager` | 8500; 8501; 10000-30000 | TCP
+`Backing Image Manager` | 8000; 8001 | TCP
+`Backing Image Data Source` | 8001; 8002 | TCP
```

Comment:
Could you point me to the use pf 8002? Thank you

Reply:
It's used by the sync server/receiver of volume export:
https://github.com/longhorn/backing-image-manager/blob/455f018e5a80943c5d93a5f216ea973740f9fcee/pkg/datasource/server.go#L22-L26